tests/system/Cache/Handlers/RedisHandlerTest.php

```diff
@@ -198,6 +198,15 @@ public function testGetCacheInfo()
         $this->assertIsArray($this->handler->getCacheInfo());
     }
 
+    public function testGetMetadataNotNull(): void
+    {
+        $this->handler->save(self::$key1, 'value');
+
+        $metadata = $this->handler->getMetaData(self::$key1);
```

Comment:
Please add an empty line.
Arrange-Act-Assert.
See https://automationpanda.com/2020/07/07/arrange-act-assert-a-pattern-for-writing-good-tests/